python_ta/reporters/templates/script.js

```diff
@@ -13,7 +13,7 @@ $(".sidebar button").click(function () {
  * https://carlanderson.xyz/how-to-animate-on-height-auto/
  */
 function toggleElement(elem) {
-  const expanded = elem.classList.contains("expanded")
+  const expanded = elem.classlist.contains("expanded")
```

Comment:
This file also shouldn't have been changed at all; more generally, only `.py` files (and the documentation files) should be changed. Revert all other changes